mandatory/2-function-creation.js

```diff
@@ -4,7 +4,13 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
+  if (userAge >= 18 && isLoggedIn === true) {
```

Comment:
in this case, you can just say 
`if (userAge >= 18 && isLoggedIn)`
for cleaner code :)

Reply:
Sure. Thank you for your guidance.